src/app/containers/MediaPlayer/MediaPlayerMetadata.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import { shape } from 'prop-types';
+import getMediaPlayerMetadata from './helpers/getMediaPlayerMetadata';
+
+const MediaPlayerMetadata = ({ aresMediaBlock }) => {
```

Comment:
I don't think this function name or file needed to change as it's nested within the `MediaPlayer/` directory. Sorry for any confusion. Please see https://github.com/bbc/simorgh/pull/4339#discussion_r336961664.